yarn.lock

```diff
@@ -1,13513 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't think that this file should be deleted. It's not a temporary .lock file, it's rather an config file to freeze the installed packages and make the build reproducible.
https://classic.yarnpkg.com/en/docs/yarn-lock/